lib/cdo/script_config.rb

```diff
@@ -16,4 +16,19 @@ def self.allows_public_caching_for_script(script_name)
   def self.cached_scripts
     HttpCache.cached_scripts
   end
+
+  # Returns the names of the scripts affected by the 'Scale Mode' feature setting.
+  def self.scale_mode_scripts
+    cached_scripts + %w(
+      20-hour
+      course1
+      course2
+      course3
+      course4
+      playlab
+      artist
+      infinity
+      iceage
```

Comment:
those are all included in this array via `cached_scripts`. The constant list here only includes scripts that are not already 'cached scripts' (which are all assumed to be 'scale mode') that also need to be marked as 'scale mode' for feature flag management.